src/components/ExpandedNode/ExpandedNode.tsx

```diff
@@ -44,7 +44,7 @@ const ExpandedNode: FC<ExpandedNodeProps> = ({
   const { patientRecords, setPatientRecords } = usePatientRecords();
   const client = useFHIRClient();
   const note = useNote();
-  const patient = usePatient();
+  const patient = usePatient().patient as fhir.Patient;
```

Comment:
The type of `usePatient().patient` is `fhir.Patient | null`. I think all this `as fhir.Patient` is doing is covering a null check. In my personal opinion we should make the null check explicit because it is easier to read but would like to hear from others too.

Reply:
I mean by the time we reach these components, `patient` will have already been set since the app will produce a loading screen instead of rendering the `children` components when `patient` is `null`.